src/main/java/core/basesyntax/dao/FruitDaoImpl.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exeption.FruitShopExeption;
+
+public class FruitDaoImpl implements FruitDao {
```

Comment:
Dao shouldn't contain any business logic, like how to add or how to remove. For this task it's enough to have only 2 method: `put(String, Integer)` that should put new value or rewrite existing, and `get(String)` that just return amount by key